js/views/modals/Settings/Advanced.js

```diff
@@ -0,0 +1,125 @@
+import app from '../../../app';
+import loadTemplate from '../../../utils/loadTemplate';
+import baseVw from '../../baseVw';
+import $ from 'jquery';
+
+const siblingsToggle = '[data-next-siblings-toggle]';
+const siblingsToggleInputs = `${siblingsToggle} input[type="radio"]`;
+
+function getResponseReason(resp) {
```

Comment:
In the code I gave you, this same thing was accomplished in one line:

```
resp && resp.responseJSON && resp.responseJSON.reason || '';
```

Why break it up into a 9 line function?